### PR TITLE
[OSDEV-744] API. When user want to confirm/reject potential_match it didn't found a match through 'id'

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-744](https://opensupplyhub.atlassian.net/browse/OSDEV-744) API. When user want to confirm/reject potential_match it didn't found a match through 'id'
- Got match id from live/origin table when create equal true for Potential Match confirm url

### CODE REVIEW

File: src/django/api/processing.py [hunk 1]
```diff
@@ -639,13 +639,21 @@ def handle_external_match_process_result(id, result, request, should_create):
...
         # No Match and Geocoder Returned No Results
         return get_error_match_result(f_l_item.id, result)
-    if (queryset_f_m.count() == 1
-            and queryset_f_m[0].status == FacilityMatchTemp.AUTOMATIC):
+
+    automatic_matches = [
+        match for match in queryset_f_m
+        if match.status == match_object_type.AUTOMATIC
+    ]
+    if (len(automatic_matches) == 1):
```
Comment (VadimKovalenkoSNF) [inline src/django/api/processing.py:656]: Question: If there is at least 1 automatic match here, shall we operate with new facility here at all? I mean, why do we need `# New Facility` block here?

**Review by VadimKovalenkoSNF — COMMENTED** [review]
Left a question

File: src/django/api/processing.py (hunk 1 above)
Comment (roman-stolar) [reply src/django/api/processing.py:656]: @VadimKovalenkoSNF Yes, Automatic Match status apply for both situations **NEW_FACILITY** & **MATCHED**.

**Review by VadimKovalenkoSNF — COMMENTED** [review]
Approved

**Review by VadimKovalenkoSNF — APPROVED** [review]

**Review by mazursasha1990 — APPROVED** [review]
Approve

**Pull request merged**